### PR TITLE
add-people-table

### PR DESCRIPTION
**Pull request by DidenkoDenis306**
[DEMO LINK](https://DidenkoDenis306.github.io/react_people-table-basics/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,172 +1,30 @@
...
+        <Routes>
+          <Route path="people">
+            <Route index element={<PeoplePage />} />
+            <Route path=":slug" element={<PeoplePage />} />
+          </Route>
+
+          <Route path="home" element={<Navigate to="/" replace />} />
+          <Route path="/" element={<HomePage />} />
+          <Route path="*" element={<NotFoundPage />} />
+        </Routes>
```
Comment (33Orange) [inline src/App.tsx:27]: Create separate component `AppRouter` for this

File: src/App.tsx [hunk 2]
```diff
@@ -1,172 +1,30 @@
...
-          >
-            People
-          </a>
-        </div>
-      </div>
-    </nav>
+    <Navigation />
 
     <main className="section">
       <div className="container">
```
Comment (33Orange) [inline src/App.tsx:17]: Is your router inside a section?

Let's move inside a router.
Create a `Layout` component and wrap all your routes in the `Layout` route.

```
<Routes>
       <Route path="/" element={<Layout/>}>
             ...
        </Route>
</Routes>
```
In Layout, you can use `Outlet` from `react-router-dom` to render your components inside.

File: src/components/PageNavLink/PageNavLink.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import cn from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+interface Props {
+  to: string;
+  title: string;
+}
+
+export const PageNavLink: React.FC<Props> = ({ to, title }) => {
```
Comment (33Orange) [inline src/components/PageNavLink/PageNavLink.tsx:10]:
```suggestion
export const NavigationLink: React.FC<Props> = ({ to, title }) => {
```

**Review by 33Orange — CHANGES_REQUESTED** [review]
Almost done

**Review by Shevtsooov — APPROVED** [review]
Well done! 🔥

File: src/components/PeopleTable/PeopleTable.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            <td>
+              <PersonLink person={person} />
+            </td>
+
+            <td>{person.sex}</td>
+            <td>{person.born}</td>
+            <td>{person.died}</td>
+            <td>{getParent(person.motherName)}</td>
+            <td>{getParent(person.fatherName)}</td>
+          </tr>
```
Comment (dffUqp) [inline src/components/PeopleTable/PeopleTable.tsx:56]: create separate component for this

**Review by dffUqp — CHANGES_REQUESTED** [review]
Good work but needs a few fixes to make it better😃

**Pull request closed without merging**